.github/workflows/smoke-test.yml

```
@@ -0,0 +1,83 @@
+name: Smoke test
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *" # Every day at 00:00 UTC
```

As agreed with you and @mszostok, let's go with 3h first 👍 
```suggestion
    - cron: "0 */3 * * *" # Every 3 hours
```